src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { TodoItem } from '../TodoItem';
+import { Todo } from '../../types';
+
+interface Props {
+  todos: Todo[];
+  onDelete: (todoToDelete: Todo) => void;
+  tempTodo: Todo | null;
+}
+
+export const TodoList: React.FC<Props> = React.memo(({
```

Comment:
I believe, looks tidier
```suggestion
export const TodoList: FC<Props> = memo(({
```